actionview/lib/action_view/renderer/abstract_renderer.rb

```diff
@@ -170,12 +170,6 @@ def extract_details(options) # :doc:
         details || NO_DETAILS
       end
 
-      def instrument(name, **options) # :doc:
```

Comment:
The doc indicates that this may be public API. Maybe a GitHub search would reveal if its used?

Reply:
I was confused by this. This is `:doc:`, but the whole class is `:nodoc:`, which seems to take precedence (I don't see docs generated for this method).

I'd love to remove if possible, but it would also be really easy to deprecate.

Reply:
This `# :doc:` is added at 7f998540af7c3398a3dae760e3e5f0218dec8897, systematically change `protected` methods to `private` methods with `# :doc:`. This `# :doc:` is accidentally added even though originally nodoc method, so we can just remove this method I think.

Reply:
Ah, perfect. 👍 to remove it then